### PR TITLE
Updates to Lesson 0, including example formatting for values in priva…

### PR DESCRIPTION
**Pull request by gwsii**
…te.yml and suggesting that they use NVM as a way to install different Node JS version SxS.

### CODE REVIEW

File: Lesson0_BeforeWorkshop/README.md [hunk 1]
```diff
@@ -6,9 +6,10 @@ Setup nordstrom.slack.com and subscribe to the #serverless-discuss channel, this
 
 ### Step 1: Install node.js
 Ensure that you have [Node.js](https://nodejs.org/en/) (v4 or better) installed.
+We suggest using [NVM](https://github.com/creationix/nvm/blob/master/README.markdown) to allow side-by-side install of different node versions.
 
-### Step 2: Ensure that you have access to an AWS Account.
-If you are a Nordstrom engineer, you may want to use a Nordstrom Public Cloud V2 account.
+### Step 2: Ensure that you have access to a Public Cloud V2 (PCV2) AWS Account.
+If you are a Nordstrom engineer, you'll want to use a Nordstrom Public Cloud V2 account.
 Nordstrom Public Cloud V2 is a substantial improvement to Pub Cloud V1.  It gives you far greater access and control to all of the things.  This workshop will not work on a public cloud v1 account.  If you are not a Nordstrom engineer and you're reading this a generic AWS account will work just fine.  If you're at a company that limits your ability to create roles/buckets/API Gateways your mileage will vary.
```
Comment (gwsii) [inline Lesson0_BeforeWorkshop/README.md]: So during our meeting over coffee this morning. We discussed the fact that allowing non-PCV2 accounts, and/or allowing participants to use personal accounts will complicate the process. If we want to simplify the steps taken to get things working and have a higher degree of confidence that they will be able to complete the steps IMO we should focus on PCV2. Is it a priority to make this workshop more generally accessible outside the Nordstrom development community? I think it should be, ultimately, yes. Are we comfortable including details about internal Nordstrom technology on GitHub?

File: Lesson0_BeforeWorkshop/README.md [hunk 2]
```diff
@@ -60,6 +61,26 @@ $ export AWS_SESSION_TOKEN=<session-token>             # this one is optional
...
+
+# deploymentBucket: #<optional S3 bucket> # uncomment the use of this variable in your serverless.yml files to deploy to a specific bucket
+
+# Core Stream
+coreStream:
+  accountId: 8888888888888
+  awslabsRoleArn: arn:aws:iam::${self:custom.private.coreStream.accountId}:role/fanoutRole
+
+```
+
```
Comment (gwsii) [inline Lesson0_BeforeWorkshop/README.md]: Felt that having an example of the formats we expect in private.yml would be helpful.

File: Lesson5_CleanUpYourMess/README.md [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+
+```sh
+serverless remove -s $STAGE
+```
+
+You should see a message from Serverless when you stack is removed.
+
+## remove ingress-stream
+
+_*** IMPORTANT! ***_
```
Comment (gwsii) [inline Lesson5_CleanUpYourMess/README.md]: Really wish MD had some sort of <blink> tag for this....

File: Lesson0_BeforeWorkshop/README.md (hunk 1 above)
Comment (erikerikson) [inline Lesson0_BeforeWorkshop/README.md]: this is made for public consumption.

File: Lesson0_BeforeWorkshop/README.md (hunk 2 above)
Comment (erikerikson) [inline Lesson0_BeforeWorkshop/README.md]: Let's put those in the file they will be editing then?

File: Lesson0_BeforeWorkshop/README.md [hunk 4]
```diff
@@ -83,3 +104,15 @@ From your workshop directory:
 $ git clone https://github.com/Nordstrom/hello-retail-workshop.git
 ```
 For more information on using github, go to https://help.github.com/articles/fork-a-repo/
+
+### Step 7: choose a unique $STAGE name and set the $REGION for your deployed services
+
+We recommend you use your LAN ID to ensure it's unique, but you can use any name you want. Go ahead and set a shell variable to use later on:
```
Comment (erikerikson) [inline Lesson0_BeforeWorkshop/README.md]: LAN ID is a Nordstrom specific thing

File: Lesson0_BeforeWorkshop/README.md [hunk 5]
```diff
@@ -83,3 +104,15 @@ From your workshop directory:
...
+
+```sh
+export STAGE=b0bb
```
Comment (erikerikson) [inline Lesson0_BeforeWorkshop/README.md]: nice improvement

File: Lesson3_PublicEndpointToAccessView/README.md [hunk 6]
```diff
@@ -14,27 +14,71 @@ More details can be found here: https://serverless.com/framework/docs/providers/
...
+
+```
+https://zzzzzzzzzz.execute-apic.us-west-2.amazonaws.com/XXXX
+```
+
+Where `zzzzzzzzzzzz` is generated by AWS and `XXXX` will be the name of the stage, and the
+
 ```sh
-/scores?role=<creator|photographer>&limit=2
+wget -qO- https://iytaw7zggd.execute-api.us-west-2.amazonaws.com/a09y/scores?role=creator&limit=2 | more
```
Comment (erikerikson) [inline Lesson3_PublicEndpointToAccessView/README.md]: perhaps the endpoint should be defined as a ENV var?  Also, this is a real one and has a LAN ID

File: Lesson3_PublicEndpointToAccessView/README.md [hunk 7]
```diff
@@ -14,27 +14,71 @@ More details can be found here: https://serverless.com/framework/docs/providers/
...
 
 ### Step 4: confirm that the AWS API Gateway deployed
 * Look in the AWS console under AWS API Gateway.
 * Copy the URL you find there under Stages.  That is the root of your endpoint.
 * Go to that URL in your browser with the following paths.
 
+Here's an example of what the URL looks like for an API Gateway endpoint:
+
+```
+https://zzzzzzzzzz.execute-apic.us-west-2.amazonaws.com/XXXX
```
Comment (erikerikson) [inline Lesson3_PublicEndpointToAccessView/README.md]: ".execute-api`c`"?

Comment (erikerikson) [inline Lesson3_PublicEndpointToAccessView/README.md]: perhaps the script that grabs the id below (line 72) should precede this?

File: Lesson5_CleanUpYourMess/README.md [hunk 8]
```diff
@@ -2,6 +2,34 @@
 
 Innovate, experiment, but flush when your're done! Don't leave unused resources in your team's AWS accounts.  The Lambdas and API Gateways will only cost you money while used, but the Kinesis streams and DynamoDB tables will charge part of their cost just for existing.  Delete everything when you're done!
 
-https://serverless.com/framework/docs/providers/aws/cli-reference/remove/#aws---remove
+## remove winner-api
 
-//todo: add instructions to serverless remove all resources
+In the Lesson 2 directory, run:
```
Comment (erikerikson) [inline Lesson5_CleanUpYourMess/README.md]: lesson 2?

File: Lesson5_CleanUpYourMess/README.md [hunk 9]
```diff
@@ -2,6 +2,34 @@
...
+
+_*** IMPORTANT! ***_
+
+BEFORE REMOVING YOUR STREAM, BE SURE TO INFORM THE WORKSHOP INSTRUCTOR AND PROVIDE YOUR STREAM ARN.
+
+If you do not, it is possible that backups can occur with the fan-out stream.
+
+Once you've been given the all-clear please delete your stream.
+
+In the Lesson 3 directory, run:
```
Comment (erikerikson) [inline Lesson5_CleanUpYourMess/README.md]: lesson 3?

**Pull request merged**